src/src/pages/capabilities/KafkaCluster/Topic.js

```diff
@@ -42,13 +42,14 @@ function TopicHeader({
     }
   };
 
-  const notProvisioned = "Provisioned".toUpperCase() !== status?.toUpperCase();
+  const provisioned = "Provisioned".toUpperCase() === status?.toUpperCase();
+  const notProvisioned = !provisioned;
```

Comment:
Could you make an enum for these statuses similar to how it was done for MessageStatus inside of messagecontracts

Reply:
Sure thing.

I just followed the existing structure of this code 🤷‍♂️